package.json

```diff
@@ -2,11 +2,44 @@
   "name": "heimdalfrontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.jsx",
   "scripts": {
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack-dev-server --config webpack.dev.js",
```

Comment:
Hello @edogbosunny, nice work so far. Please change this start to start:dev since it is using webpack-dev-server